app/models/patient.rb

```diff
@@ -162,6 +164,28 @@ def event_params
     }
   end
 
+  def confirm_unique_phone_number
+    ##
+    # This method is preferred over Rail's built-in uniqueness validator
+    # so that case managers get a meaningful error message when a patient
+    # exists on a different line than the one the volunteer is serving.
+    #
+    # See https://github.com/DCAFEngineering/dcaf_case_management/issues/825
+    ##
+    already_taken_patient = Patient.where(primary_phone: primary_phone)
+    if already_taken_patient.exists?
+      # phones are unique, so there should only be one match in array
+      patients_line = already_taken_patient[0][:line]
+      volunteers_line = line
+
+      if volunteers_line == patients_line
+        errors.add(:this_phone_number_is_already_taken, "on this line.")
+      else
+        errors.add(:this_phone_number_is_already_taken, "on the #{patients_line}. Please contact the case manager on duty at the #{volunteers_line} line to confirm which line the patient should be under. If the line needs to be changed, please contact the CM Directors.")
+      end
+    end
+  end
```

Comment:
I forget whether or not this is part of the callbacks behavior, but you MAY need to `throw(:abort`) on failure to prevent saving. Something to watch out for -- I forget exactly how this works.